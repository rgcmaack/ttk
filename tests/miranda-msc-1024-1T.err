[32;1m[MorseSmaleComplex] [0m[33;1m[WARNING][0m No pre-existing order for array:
[32;1m[MorseSmaleComplex] [0m[33;1m[WARNING][0m   `ImageFile`.
[32;1m[MorseSmaleComplex] [0m[33;1m[WARNING][0m TIP: run `ttkArrayPreconditioning` first
[32;1m[MorseSmaleComplex] [0m[33;1m[WARNING][0m for improved performances :)
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
[0m[1m[91m
Loguru caught a signal: SIGABRT
[0m[0m[31mStack trace:
60      0x55ad35e7991e /usr/local/bin/pvpython(+0x291e) [0x55ad35e7991e]
59      0x2ba9c31d40b3 __libc_start_main + 243
58      0x55ad35e7a2fb /usr/local/bin/pvpython(+0x32fb) [0x55ad35e7a2fb]
57      0x2ba9c4f7bcfb vtkPythonInterpreter::PyMain(int, char**) + 491
56      0x2ba9c5e15aa7 Py_Main + 71
55      0x2ba9c5e15558 Py_RunMain + 2008
54      0x2ba9c5e34a47 PyRun_SimpleFileExFlags + 359
53      0x2ba9c5e330dc /lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x18c0dc) [0x2ba9c5e330dc]
52      0x2ba9c5e7263f PyEval_EvalCode + 31
51      0x2ba9c5e72252 PyEval_EvalCodeEx + 66
50      0x2ba9c5e71ecb _PyEval_EvalCodeWithName + 2299
49      0x2ba9c5d1d46d _PyEval_EvalFrameDefault + 4861
48      0x2ba9c5d1bd6d /lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d) [0x2ba9c5d1bd6d]
47      0x2ba9c5f4f0f4 _PyFunction_Vectorcall + 148
46      0x2ba9c5e71ecb _PyEval_EvalCodeWithName + 2299
45      0x2ba9c5d23ef6 _PyEval_EvalFrameDefault + 32134
44      0x2ba9c5d1bd6d /lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x74d6d) [0x2ba9c5d1bd6d]
43      0x2ba9c5f4fd8a /lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x2a8d8a) [0x2ba9c5f4fd8a]
42      0x2ba9c5f4f0f4 _PyFunction_Vectorcall + 148
41      0x2ba9c5e71ecb _PyEval_EvalCodeWithName + 2299
40      0x2ba9c5d24f48 _PyEval_EvalFrameDefault + 36312
39      0x2ba9c5f4f667 /lib/x86_64-linux-gnu/libpython3.8.so.1.0(+0x2a8667) [0x2ba9c5f4f667]
38      0x2ba9e10e877c /usr/local/lib/python3.8/site-packages/paraview/modules/vtkRemotingViews.so(+0x1a477c) [0x2ba9e10e877c]
37      0x2ba9c70e0bc9 vtkSMParaViewPipelineControllerWithRendering::Show(vtkSMSourceProxy*, int, vtkSMViewProxy*, char const*) + 249
36      0x2ba9c70e0156 vtkSMParaViewPipelineControllerWithRendering::UpdatePipelineBeforeDisplay(vtkSMSourceProxy*, int, vtkSMViewProxy*) + 246
35      0x2ba9c4ac5a30 vtkSMSourceProxy::UpdatePipeline(double) + 144
34      0x2ba9c4a4ab3b vtkSMOutputPort::UpdatePipelineInternal(double, bool) + 203
33      0x2ba9c499d648 vtkPVSessionBase::ExecuteStream(unsigned int, vtkClientServerStream const&, bool) + 56
32      0x2ba9c499e7b4 vtkPVSessionCore::ExecuteStream(unsigned int, vtkClientServerStream const&, bool) + 84
31      0x2ba9c499e986 vtkPVSessionCore::ExecuteStreamInternal(vtkClientServerStream const&, bool) + 246
30      0x2ba9c4f954c5 vtkClientServerInterpreter::ProcessStream(vtkClientServerStream const&) + 37
29      0x2ba9c4f95002 vtkClientServerInterpreter::ProcessOneMessage(vtkClientServerStream const&, int) + 194
28      0x2ba9c4f94ec5 vtkClientServerInterpreter::ProcessCommandInvoke(vtkClientServerStream const&, int) + 1173
27      0x2ba9c4f94897 vtkClientServerInterpreter::CallCommandFunction(char const*, vtkObjectBase*, char const*, vtkClientServerStream const&, vtkClientServerStream&) + 503
26      0x2ba9c37c90e3 vtkSISourceProxyCommand(vtkClientServerInterpreter*, vtkObjectBase*, char const*, vtkClientServerStream const&, vtkClientServerStream&, void*) + 2243
25      0x2ba9c49d6232 vtkSISourceProxy::UpdatePipeline(int, double, bool) + 514
24      0x2ba9c52e7587 vtkStreamingDemandDrivenPipeline::Update(int, vtkInformationVector*) + 279
23      0x2ba9c529c391 vtkDemandDrivenPipeline::UpdateData(int) + 161
22      0x2ba9c52e61bd vtkStreamingDemandDrivenPipeline::ProcessRequest(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 893
21      0x2ba9c529d516 vtkDemandDrivenPipeline::ProcessRequest(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 310
20      0x2ba9c5296512 vtkCompositeDataPipeline::ForwardUpstream(vtkInformation*) + 386
19      0x2ba9c52e61bd vtkStreamingDemandDrivenPipeline::ProcessRequest(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 893
18      0x2ba9c529d91f vtkDemandDrivenPipeline::ProcessRequest(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 1343
17      0x2ba9c5297721 vtkCompositeDataPipeline::ExecuteData(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 129
16      0x2ba9c529acbd vtkDemandDrivenPipeline::ExecuteData(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 61
15      0x2ba9c52a0c46 vtkExecutive::CallAlgorithm(vtkInformation*, int, vtkInformationVector**, vtkInformationVector*) + 86
14      0x2ba9dd215f28 ttkAlgorithm::ProcessRequest(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 1364
13      0x2ba9db6eed98 ttkMorseSmaleComplex::RequestData(vtkInformation*, vtkInformationVector**, vtkInformationVector*) + 2260
12      0x2ba9db8199ea int ttkMorseSmaleComplex::dispatch<float, ttk::ImplicitTriangulation>(vtkDataArray*, vtkPolyData*, vtkPolyData*, vtkPolyData*, long long const*, ttk::ImplicitTriangulation const&) + 160
11      0x2ba9db818a0e int ttk::MorseSmaleComplex::execute<float, ttk::ImplicitTriangulation>(ttk::MorseSmaleComplex::OutputCriticalPoints&, ttk::MorseSmaleComplex::Output1Separatrices&, ttk::MorseSmaleComplex::Output2Separatrices&, ttk::MorseSmaleComplex::OutputManifold&, float const*, unsigned long, long long const*, ttk::ImplicitTriangulation const&) + 288
10      0x2ba9da5ebb68 int ttk::dcg::DiscreteGradient::buildGradient<ttk::ImplicitTriangulation>(ttk::ImplicitTriangulation const&) + 364
9       0x2ba9de9a5143 ttk::dcg::DiscreteGradient::initMemory(ttk::AbstractTriangulation const&) + 429
8       0x2ba9c551a717 std::vector<long long >::_M_fill_insert(__gnu_cxx::__normal_iterator<long long*, std::vector<long long > >, unsigned long, long long const&) + 871
7       0x2ba9c6afd522 /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e522) [0x2ba9c6afd522]
6       0x2ba9c6b096a9 /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa6a9) [0x2ba9c6b096a9]
5       0x2ba9c6b093f7 /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7) [0x2ba9c6b093f7]
4       0x2ba9c6b0938c /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c) [0x2ba9c6b0938c]
3       0x2ba9c6afd911 /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911) [0x2ba9c6afd911]
2       0x2ba9c31d2859 abort + 299
1       0x2ba9c31f303b gsignal + 203
0       0x2ba9c31f30c0 /lib/x86_64-linux-gnu/libc.so.6(+0x430c0) [0x2ba9c31f30c0][0m
[0m[31m(1752.894s) [paraview        ]                       :0     FATL| Signal: SIGABRT[0m
/var/spool/slurmd/job5189248/slurm_script: line 4: 87320 Aborted                 singularity exec -B /groups/VIS/data2:/data ~/ttk-private.sif pvpython miranda.py msc 1024 1
