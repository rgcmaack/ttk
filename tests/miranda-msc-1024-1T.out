miranda benchmark msc, size 1024, 1 threads
[32;1m[Common] [0m[32;1m                           [0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m
[32;1m[Common] [0m[32;1m                    ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m
[32;1m[Common] [0m[32;1m                [0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m
[32;1m[Common] [0m[32;1m             ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░[0m[2;1m░[0m[2;1m▒░░░▒▒░[0m[32;1m░[0m[32;1m░[0m[32;1m▒▒▒▒▒▒▒[0m
[32;1m[Common] [0m[32;1m          ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[2;1m░[0m[30;1m░[0m[30;1m░░░[0m[2;1m░   [0m[34;1m░[0m[34;1m░░░░░░[0m[30;1m░[0m[30;1m░░░▒[0m[2;1m▒[0m[32;1m▒[0m[32;1m▒▒▒▒[0m
[32;1m[Common] [0m[32;1m        ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░[0m[30;1m░                    [0m[34;1m░[0m[34;1m░░[0m[30;1m░[0m[30;1m░░[0m[32;1m░[0m[32;1m▒▒▒[0m
[32;1m[Common] [0m[32;1m      ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[2;1m░[0m[30;1m░    [0m[30;1m▒▒▒▒▒▒▒▒▒▒▒▒          [0m[34;1m░[0m[34;1m▒▒ [0m[30;1m░[0m[2;1m░[0m[32;1m▒[0m[32;1m▒▒[0m
[32;1m[Common] [0m[32;1m     ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m[30;1m░[0m[30;1m░   ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░▒▒       [0m[34;1m░[0m[34;1m░▒ [0m[30;1m░[0m[30;1m░[0m[32;1m▒[0m[32;1m▒[0m
[32;1m[Common] [0m[32;1m    ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m[30;1m░[0m[30;1m░  [0m[2;1m▒[0m[2;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m[30;1m▒[0m[30;1m▒▒▒▒░▒▒    [0m[34;1m░[0m[34;1m░░░▒ [0m[30;1m░[0m[2;1m░[0m[32;1m▒[0m[32;1m▒[0m
[32;1m[Common] [0m[32;1m   ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m[30;1m░[0m[30;1m░ ▒[0m[2;1m▒[0m[2;1m▒▒▒▒▒[0m[2;1m▒[0m[2;1m▒▒▒[0m[37;1m▒[0m[37;1m▄[0m[37;1m▓█[0m[37;1m█▓████[0m[37;1m█▓▄[0m[2;1m▄[0m[2;1m░[0m[30;1m░[0m[30;1m░▒   [0m[34;1m░[0m[34;1m░░░░░[0m[30;1m░[0m[30;1m░[0m[32;1m░[0m[32;1m▒[0m
[32;1m[Common] [0m[32;1m  ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░  [0m[2;1m▒[0m[2;1m▒▒▒[0m[2;1m▒[0m[2;1m▒▒▒▒▓[0m[37;1m▓[0m[37;1m▓▀[0m[2;1m▀[0m[2;1m▀▀[0m[37;1m▀[0m[37;1m▀[0m[37;1m▀████▓▓▓▓[0m[2;1m▄[0m[30;1m░[0m[30;1m░   [0m[34;1m░[0m[34;1m░░░░▒ [0m[30;1m░[0m[2;1m░[0m[32;1m▒[0m
[32;1m[Common] [0m[32;1m ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m░[0m[30;1m░ [0m[30;1m▒[0m[2;1m▒[0m[2;1m▒▒[0m[2;1m▒[0m[2;1m▒▒▒▒           [0m[30;1m░[0m[30;1m░[0m[2;1m░[0m[37;1m▀[0m[37;1m██▓▓▓▓[0m[2;1m▓[0m[30;1m░[0m[2;1m░    [0m[34;1m░[0m[34;1m░░▒ [0m[30;1m░[0m[2;1m░[0m[32;1m▒[0m
[32;1m[Common] [0m[32;1m ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒[0m[32;1m░[0m[30;1m░ [0m[30;1m▒[0m[2;1m▒[0m[2;1m▒▒▒▒                  [0m[30;1m░[0m[2;1m▀[0m[37;1m█[0m[37;1m█▓▓▓[0m[37;1m▓[0m[2;1m░    [0m[34;1m░[0m[34;1m░░░▒ [0m[30;1m░[0m[32;1m░[0m[32;1m░[0m
[32;1m[Common] [0m[32;1m ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒[0m[32;1m░[0m[32;1m▒[0m[30;1m░[0m[30;1m░ [0m[2;1m▒[0m[2;1m▒▒                     [0m[30;1m░[0m[30;1m░[0m[37;1m▀[0m[37;1m▓[0m[37;1m▓▓[0m[37;1m▓▒[0m[2;1m░    [0m[34;1m░[0m[34;1m░░▒░[0m[30;1m░[0m[30;1m░[0m[32;1m▒[0m
[32;1m[Common] [0m[32;1m ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░[0m[30;1m░ [0m[30;1m▒▒                        ░[0m[2;1m▐[0m[37;1m▓[0m[37;1m▓[0m[37;1m▓▒[0m[37;1m▒    [0m[34;1m░[0m[34;1m░░░▒ [0m[30;1m░[0m[32;1m░[0m
[32;1m[Common] [0m[32;1m ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░                         [0m[32;1m░[0m[30;1m░[0m[30;1m░[0m[37;1m▐[0m[37;1m▓[0m[37;1m▓[0m[37;1m▒[0m[2;1m▒[0m[2;1m▒ [0m[34;1m░[0m[34;1m░░░░▒▒ [0m[30;1m░[0m[2;1m░[0m[32;1m▒[0m
[32;1m[Common] [0m[30;1m░[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░                        ░░[0m[30;1m░[0m[2;1m▐[0m[37;1m▓[0m[37;1m▒[0m[37;1m▒[0m[2;1m▒[0m[30;1m░[0m[34;1m░[0m[34;1m░░░░░▒▒ [0m[30;1m░[0m[30;1m░[0m[32;1m▒[0m
[32;1m[Common] [0m[30;1m░[0m[32;1m░[0m[32;1m░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░                   ░░░░░[0m[30;1m░[0m[2;1m▐[0m[37;1m▓[0m[37;1m▒[0m[2;1m▒[0m[2;1m▒[0m[34;1m░[0m[34;1m░░░░░░▒▒ [0m[30;1m░[0m[30;1m░[0m
[32;1m[Common] [0m[32;1m  ░[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░           ░░░░░░▒[0m[30;1m▒[0m[30;1m░[0m[37;1m▓[0m[37;1m▒[0m[2;1m▒[0m[2;1m▒[0m[30;1m░[0m[34;1m░[0m[34;1m░░░░░▒▒▒░[0m[30;1m░[0m[32;1m░[0m
[32;1m[Common] [0m[32;1m   [0m[30;1m░[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░[0m[30;1m░ [0m[37;1m▓[0m[2;1m▒[0m[2;1m▒[0m[30;1m░[0m[34;1m░[0m[34;1m░░░░▒▒▒▒▒[0m[30;1m░[0m[30;1m░[0m[32;1m░[0m
[32;1m[Common] [0m[32;1m    [0m[30;1m░[0m[32;1m░[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒▒[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░░░░░░▒░[0m[30;1m░[0m[30;1m░[0m[37;1m▓[0m[2;1m▒[0m[2;1m▒[0m[30;1m░[0m[34;1m▒[0m[34;1m░░▒▒▒▒▒▒▒░[0m[30;1m░[0m[30;1m░[0m
[32;1m[Common] [0m[32;1m     ░░[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒▓▓▓▒▒[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░ [0m[2;1m▓[0m[2;1m▒[0m[30;1m░ [0m[34;1m▒[0m[34;1m▒▒▒▒▒▒▒▒▒▒[0m[30;1m░[0m[30;1m░[0m[32;1m▒[0m
[32;1m[Common] [0m[32;1m      [0m[34;1m▒[0m[30;1m░[0m[2;1m░[0m[32;1m░[0m[32;1m▒▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒[0m[37;1m▓[0m[37;1m▓▓[0m[32;1m▓[0m[32;1m▒[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░[0m[30;1m░ [0m[37;1m▒[0m[2;1m▒  [0m[34;1m▒[0m[34;1m▒▒▒▒▒▒▒▒▒▒[0m[30;1m░[0m[30;1m░[0m[32;1m░[0m
[32;1m[Common] [0m[32;1m      [0m[34;1m░[0m[34;1m░░[0m[30;1m░[0m[30;1m░[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░[0m[30;1m░ [0m[2;1m▒[0m[2;1m▒   [0m[34;1m▒[0m[34;1m▒▒▒▒▒▒▒▒▒▒[0m[30;1m░[0m[30;1m░[0m[32;1m░[0m
[32;1m[Common] [0m[32;1m       [0m[34;1m░[0m[34;1m░░░░[0m[30;1m░[0m[2;1m░[0m[32;1m░[0m[32;1m▒▒▒▒▒▒▒▒[0m[32;1m▒[0m[32;1m▒[0m[32;1m▒[0m[32;1m▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░[0m[30;1m░░[0m[2;1m░[0m[2;1m▒     [0m[34;1m▒[0m[34;1m▒▒▒▒▒▒▒▒▒▒[0m[30;1m░[0m[30;1m░[0m[32;1m░[0m
[32;1m[Common] [0m[32;1m         [0m[34;1m▒[0m[34;1m░░░░░[0m[30;1m░[0m[30;1m░[0m[2;1m░[0m[32;1m░[0m[32;1m░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░[0m[2;1m░[0m[30;1m░[0m[30;1m░░ [0m[2;1m░       [0m[34;1m░[0m[34;1m▒▒▒▒▒▒▒▒▒▒░[0m[30;1m░[0m[30;1m░[0m
[32;1m[Common] [0m[32;1m          [0m[34;1m░[0m[34;1m░░░░░░░░░[0m[30;1m░[0m[30;1m░░░░░░░░░░░░░░░░            [0m[34;1m▒[0m[34;1m▒▒▒▒▒▒▒▒▒░[0m[30;1m░[0m[30;1m░[0m
[32;1m[Common] [0m[32;1m            [0m[34;1m░[0m[34;1m▒░░░░░░░░░░                       ▒▒▒▒▒▒▒[0m[36;1m▒[0m[36;1m▒[0m[34;1m▒[0m[34;1m░[0m[30;1m░[0m[30;1m░[0m
[32;1m[Common] [0m[32;1m              [0m[34;1m░[0m[34;1m▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░ ░▒[0m[36;1m▒[0m[36;1m▒▒▒▒▒▒[0m[34;1m░[0m[30;1m░[0m[30;1m░[0m[30;1m░[0m
[32;1m[Common] [0m[32;1m                 [0m[34;1m░[0m[34;1m░▒░░░░░░░░░░░░░░░░░░░░░░░▒[0m[36;1m▒[0m[36;1m▒▒▒▒▒[0m[34;1m▒[0m[34;1m░[0m[30;1m░[0m[30;1m░[0m[30;1m░[0m
[32;1m[Common] [0m[32;1m                      [0m[34;1m░[0m[34;1m░▒░░░░░░░░░░░░░░░▒[0m[36;1m▒[0m[36;1m▒▒[0m[34;1m▒[0m[34;1m▒░░[0m[30;1m░[0m[30;1m░░[0m
[32;1m[Common] [0m[32;1m                              [0m[34;1m░[0m[34;1m░░░░░░░░░░[0m[30;1m░[0m[30;1m░▒░[0m
[32;1m[Common] [0m[0;1m _____ _____ _  __                    __  __    ____   ___ ____  ____[0m
[32;1m[Common] [0m[0;1m|_   _|_   _| |/ /                   / /__\ \  |___ \ / _ \___ \|___ \[0m
[32;1m[Common] [0m[0;1m  | |   | | | ' /                   | |/ __| |   __) | | | |__) | __) |[0m
[32;1m[Common] [0m[0;1m  | |   | | | . \                   | | (__| |  / __/| |_| / __/ / __/[0m
[32;1m[Common] [0m[0;1m  |_|   |_| |_|\_\                  | |\___| | |_____|\___/_____|_____|[0m
[32;1m[Common] [0m[0;1m                                     \_\  /_/[0m
[32;1m[Common] [0m[0;1mWelcome![0m
[32;1m[Common] [0m[33;1m[WARNING][0m 
[32;1m[Common] [0m[33;1m[WARNING][0m [33;1mTTK has *NOT* been built in double-templating mode!
[32;1m[Common] [0m[33;1m[WARNING][0m [33;1mDEVELOPERS ONLY!
[32;1m[Common] [0m[33;1m[WARNING][0m [33;1mExpect unsupported types for bivariate data.
[32;1m[Common] [0m[33;1m[WARNING][0m 
[32;1m[Common] [0m[33;1m[WARNING][0m [33;1mTo enable the double-templating mode, rebuild TTK with:
[32;1m[Common] [0m[33;1m[WARNING][0m [33;1m  -DTTK_ENABLE_DOUBLE_TEMPLATING=ON
[32;1m[Common] [0m[33;1m[WARNING][0m 
[32;1m[MorseSmaleComplex] [0m%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%[34;1m[0m
[32;1m[MorseSmaleComplex] [0mInitializing order array. ...................[34;1m [0.000s|1T|0%][0m[32;1m[MorseSmaleComplex] [0mInitializing order array. ...............[34;1m [892.328s|1T|100%][0m
[32;1m[MorseSmaleComplex] [0mLaunching computation on field `ImageFile'...
